invenio_communities/templates/semantic-ui/invenio_communities/macros/community_uploads.html

```diff
@@ -0,0 +1,27 @@
+{#
+Copyright (C) 2024 CERN
+
+Invenio App RDM is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% from "zenodo_rdm/macros/record_item.html" import record_item %}
```

Comment:
major: `invenio-communities` is a module that is completely isolated from an instance (e.g. `zenodo-rdm`). It is ok for instances to import modules, not the other way around. 

That being said, if you need `record_item` here then it  should be implemented in a library that `invenio-communities` depends on.